api/base/middleware.py

```diff
@@ -68,9 +68,9 @@ class CorsMiddleware(corsheaders.middleware.CorsMiddleware):
     _context = threading.local()
 
     def origin_found_in_white_lists(self, origin, url):
-        settings.CORS_ORIGIN_WHITELIST += api_settings.ORIGINS_WHITELIST
+        settings.CORS_ORIGIN_WHITELIST += api_settings.CORS_ORIGIN_WHITELIST
         # Check if origin is in the dynamic custom domain whitelist
-        found = super(CorsMiddleware, self).origin_found_in_white_lists(origin, url)
+        found = super().origin_found_in_white_lists(origin, url)
```

Comment:
Missed this one: `super()` is equivalent to `super(CorsMiddleware, self)` in this case with python3 and I like this change. Just wondering is there any other reason that led to this change?

Reply:
I'm just trying to phase in Python3 syntax slowly, but really should be doing this now most places.